i18n/en.json

```diff
@@ -1,4 +1,8 @@
 [
+  {
+    "id": "",
```

Comment:
❓ Why are you adding a translation with an empty id and translation?

Reply:
This is being caused by `return model.NewAppError("SendDelinquencyEmail", "", nil, err.Error(), http.StatusInternalServerError)`. We need to add a translation id to calls of NewAppError.

Reply:
I thought I saw a demo somewhere about NewAppError not requiring a translation anymore 🤔

Reply:
I've added a translation now, should pass next run